dashboard/src/app/login/login.component.ts

```diff
@@ -48,6 +48,8 @@ export class LoginComponent implements OnInit {
           navigator.credentials.get(respBody)
             .then(credential => {
               const requestBody = {
+                username: this.loginUsername,
+                mail: '',
```

Comment:
We should definitely remove the mail artifacts ...
Maybe next sprint?

Reply:
+1